packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -35,15 +34,21 @@ export default declare((api, options, dirname) => {
 
   const corejsVersion = rawVersion ? Number(rawVersion) : false;
 
-  if (![false, 2, 3].includes(corejsVersion)) {
+  if (corejsVersion === 2) {
     throw new Error(
-      `The \`core-js\` version must be false, 2 or 3, but got ${JSON.stringify(
+      "Since Babel 8, the core-js 2 support has been dropped. Please use 'corejs: 3'.",
+    );
+  }
+
+  if (![false, 3].includes(corejsVersion)) {
+    throw new Error(
+      `The \`core-js\` version must be false or 3, but got ${JSON.stringify(
         rawVersion,
       )}.`,
     );
   }
 
-  if (proposals && (!corejsVersion || corejsVersion < 3)) {
+  if (proposals && !corejsVersion) {
     throw new Error(
       "The 'proposals' option is only supported when using 'corejs: 3'",
```

Comment:
Maybe we should remove the version number, since `proposals` is now supported by every supported `core-js` version?

```suggestion
      "The 'proposals' option is only supported when using core-js",
```